src/BlockchainTestsFiller/bcStateTests/suicideCoinbaseFiller.json

```diff
@@ -43,6 +43,36 @@
 				"balance" : "5000000000000420000"
 			    },
 			    "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+				"balance" : "9999472208",
```

Comment:
I don't understand why this balance changed.

Reply:
changed from what?

Reply:
"9999447208",

Reply:
so there were no bytecode changes on homestead but gas usage has changed? 
lets see what other clients says about this test

Reply:
OK.

Reply:
ah wait. you fixed the cpp client from typo? could it be that some code now works on homestead rules now and thus this test is different?

Reply:
That shouldn't change the test results.